src/si/illuminance.rs

```diff
@@ -0,0 +1,61 @@
+//! Illuminance (base unit lux, lx, cd · sr / m²).
+
+quantity! {
+    /// Illuminance (base unit lux, lx, cd · sr / m²).
+    quantity: Illuminance; "illuminance";
+    /// Dimension of illuminance, E (base unit lux, lx, cd · sr / m²).
+    dimension: ISQ<
+        N2,     // length
+        Z0,     // mass
+        Z0,     // time
+        Z0,     // electric current
+        Z0,     // thermodynamic temperature
+        Z0,     // amount of substance
+        P1>;    // luminous intensity
+    kind: dyn (crate::si::marker::IlluminanceKind);
+    units {
+        @yottalux: prefix!(yotta); "Ylm", "yottalux", "yottaluxs";
+        @zettalux: prefix!(zetta); "Zlm", "zettalux", "zettaluxs";
+        @exalux: prefix!(exa); "Elm", "exalux", "exaluxs";
+        @petalux: prefix!(peta); "Plm", "petalux", "petaluxs";
+        @teralux: prefix!(tera); "Tlm", "teralux", "teraluxs";
+        @gigalux: prefix!(giga); "Glm", "gigalux", "gigaluxs";
+        @megalux: prefix!(mega); "Mlm", "megalux", "megaluxs";
+        @kilolux: prefix!(kilo); "klm", "kilolux", "kiloluxs";
+        @hectolux: prefix!(hecto); "hlm", "hectolux", "hectoluxs";
+        @decalux: prefix!(deca); "dalm", "decalux", "decaluxs";
+        /// The lux is defined to be 1 lumen per square meter,
+        /// or 1 candela steradian per square meter.
+        @lux: prefix!(none); "lm", "lux", "luxs";
```

Comment:
```suggestion
        @lux: prefix!(none); "lx", "lux", "lux";
```

Copy/paste error from luminous flux? It also seems the plural and singular are the [same](https://en.wikipedia.org/wiki/Lux).